server/routes/api/browser.py

```diff
@@ -110,3 +110,52 @@ def get_observation_ids():
         dcid = cells[0].get('value', '')
         result[obsDate] = dcid
     return Response(json.dumps(result), 200, mimetype='application/json')
+
+
+def statvar_hierarchy_helper(svg_id, sv_groups, processed_svg, processed_sv,
```

Comment:
nit: consistently using svg here? example: sv_groups -> svg_map

Reply:
Also please add a function comment on input and the behavior; Worth to mention that the svg_map is modified to add a "parent" field

Reply:
done